recipes/soplex/all/conanfile.py

```diff
@@ -90,9 +90,10 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.variables["MPFR"] = False
         tc.variables["GMP"] = self.options.with_gmp
         tc.variables["BOOST"] = self.options.with_boost
-        tc.variables["Boost_VERSION_MACRO"] = "108300"
+        tc.variables["Boost_VERSION_MACRO"] = "108400"
```

Comment:
```suggestion
        tc.variables["Boost_VERSION_MACRO"] = "108400"
        tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW"
```

The CMake policy version in the project is 3.9 and needs CMP0077 to be enabled for `tc.variables` to work correctly.

Reply:
added